Gemfile.lock

```diff
@@ -137,16 +137,10 @@ GEM
       sprockets (~> 2.2.1)
     active_model_serializers (0.8.3)
       activemodel (>= 3.0)
-    active_utils (2.2.3)
-      activesupport (>= 2.3.11)
-      i18n
-    activemerchant (1.43.3)
-      active_utils (~> 2.0, >= 2.0.1)
-      activesupport (>= 2.3.14, < 5.0.0)
+    activemerchant (1.78.0)
```

Comment:
@mkllnk @oeoeaio I think you guys mentioned something about `activemerchant` versions somewhere but I don't remember what.